protocol/activity_center.go

```diff
@@ -48,14 +49,19 @@ func (n *ActivityCenterNotification) Valid() error {
 
 }
 
-func showMentionActivityCenterNotification(publicKey ecdsa.PublicKey, message *common.Message, chat *Chat, responseTo *common.Message) bool {
-	if chat != nil && !chat.Active {
-		return false
+func showMentionOrReplyActivityCenterNotification(publicKey ecdsa.PublicKey, message *common.Message, chat *Chat, responseTo *common.Message) int {
+	if chat == nil || !chat.Active || (!chat.CommunityChat() && !chat.PrivateGroupChat()) {
```

Comment:
Just wondering, why excluding private group chats or community chats?

Reply:
I reused the same condition Brian added and put it at he start.

Unless I messed up, it should be that we only add mentions and replies from group chats and community chats.

Reply:
oh I missed that, I think we should support all kind of chats probably, I'd leave it out for now, and unless something is not working we can change it